unstructured/ingest/main.py

```diff
@@ -32,6 +32,7 @@ def __init__(
         reprocess,
         verbose,
         max_docs,
+        recursive,
```

Comment:
why do we need to pass recursive here? we ultimately set/get this via SimpleFsspecConfig, which has already been assigned on the connector at this point, right?

Reply:
Truth!